src/services/Connector.js

```diff
@@ -37,11 +37,13 @@ class Connector extends BasicConnector {
                     handler: this._blocks.getBlock,
                     scope: this._blocks,
                     validation: {
-                        required: ['blockId'],
                         properties: {
                             blockId: {
                                 type: 'string',
                             },
+                            blockNum: {
+                                type: 'number',
+                            },
```

Comment:
it's possible to setup validator to allow exactly 1 param. now it additionally accepts both or none